packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -102,6 +102,7 @@ export const buildIframeSrc = (params: BuildIframeSrcOptions): string => {
     Object.entries(options.hidden)
       .filter(([, paramValue]) => isDefined(paramValue))
       .forEach(([paramName, paramValue]) => {
+        url.searchParams.delete(paramName)
```

Comment:
What if hidden field value is empty string? It will overwrite transitive search param, correct? Or will share page not include hidden fields with empty value?

Reply:
transitive params will take priority over empty hidden field values, pushing the changes..